scania-digital-design-system/sdds
# docs(storybook): minor cleanup

**Describe pull-request**  
Minor cleanup making the code copy pasteable.

**Solving issue**  
Fixes: -

**How to test**  
1. Go to storybook
2. Check in Components -> All components -> Docs
3. Check the docs page

**Suggested test steps**
